### ISSUE TITLE
Arch: alternatives system unsupported.

### ISSUE THREAD

**Issue by noelmcloughlin** [saltstack-formulas/maven-formula#23]
There is no update-alternatives solution in Archlinux.  Isolating these alternatives.states into a separate **alternatives.sls** and updating the README is probably easiest solution.

```
------------
[DEBUG   ] Could not LazyLoad alternatives.install: 'alternatives' __virtual__ returned False
[INFO    ] Running state [maven-home] at time 17:46:55.580004
[DEBUG   ] LazyLoaded alternatives.install
[ERROR   ] State 'alternatives.install' was not found in SLS 'maven.env'
Reason: 'alternatives' __virtual__ returned False
```

**Comment by noelmcloughlin** [comment]
I'm thinking of splitting alternatives away from `env.sls ` into a new optional state and disabling alternatives by default (don't force).

**Comment by noelmcloughlin** [comment]
Resolved by #27
